modules/administration/pages/master-fingerprint.adoc

```diff
@@ -14,19 +24,23 @@ salt-key -F master
 ----
 +
 
-On your client, open the [path]``/etc/salt/minion`` configuration file.
+On your client, open the [path]``/etc/salt/minion`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
```

Comment:
For consistency, we probably should mention `/etc/salt/minion.d/custom.conf` instead of `/etc/salt/minion`:

```suggestion
On your client, open the [path]``/etc/salt/minion.d/custom.conf`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
```